slingpy/evaluation/metrics/mae.py

```diff
@@ -29,5 +29,10 @@ class MeanAbsoluteError(AbstractMetric):
     def get_abbreviation(self) -> AnyStr:
         return "MAE"
 
-    def evaluate(self, y_pred: np.ndarray, y_true: np.ndarray, threshold: Optional[float] = None) -> np.ndarray:
+    def evaluate(
+        self, y_pred: np.ndarray, y_true: np.ndarray, threshold: Optional[float] = None
+    ) -> np.ndarray:
+        if y_pred.shape[1] == 2:
```

Comment:
Mean Absolute Error should not support classification models, and raising an exception here with a classification input (shape=`(n, 2)`) is preferable (may otherwise mask errors).